test/EFCore.PG.FunctionalTests/TestUtilities/TestRelationalCommandBuilderFactory.cs

```diff
@@ -110,7 +110,7 @@ class TestRelationalCommand : IRelationalCommand
                 if (errorNumber != null)
                 {
                     connection.DbConnection.Close();
-                    throw new PostgresException { SqlState = errorNumber };
+                    throw new NpgsqlException();
```

Comment:
Can you explain these changes? They don't seem right (errorNumber just gets ignored), and I'm not sure why why they're relevant for this PR...

Reply:
Just saw your comment about this: https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/pull/917#issuecomment-507490802

We probably need to have a separate PR to port the EF Core provider over Ngpsql 4.1.